src/main/java/at/ac/fhcampuswien/fhmdb/ui/MovieCell.java

```diff
@@ -9,35 +10,45 @@
 
 public class MovieCell extends ListCell<Movie> {
 
+    private final ClickEventHandler<Movie> clickEventHandler;
+
+    public MovieCell(ClickEventHandler<Movie> clickEventHandler) {
+        super();
+        this.clickEventHandler = clickEventHandler;
+    }
+
```

Comment:
### Remove Button not shown in the Movie-Cell

Change to:
`public MovieCell(ClickEventHandler<Movie> clickEventHandler, boolean isWatchlistCell) {
        super();
        this.clickEventHandler = clickEventHandler;
        this.isWatchlistCell = isWatchlistCell;
    }`